src/components/capture/Create/index.tsx

```diff
@@ -86,46 +87,48 @@ function CaptureCreate() {
         <DetailsFormHydrator>
             <EndpointConfigHydrator>
                 <ResourceConfigHydrator>
-                    <EntityCreate
-                        entityType={entityType}
-                        draftSpecMetadata={draftSpecsMetadata}
-                        toolbar={
-                            <EntityToolbar
-                                waitTimes={{ generate: MAX_DISCOVER_TIME }}
-                                GenerateButton={
-                                    <CaptureGenerateButton
-                                        entityType={entityType}
-                                        disabled={!hasConnectors}
-                                        postGenerateMutate={updateDraftSpecs}
-                                        createWorkflowMetadata={{
-                                            initiateDiscovery,
-                                            setInitiateDiscovery,
-                                        }}
-                                    />
-                                }
-                                TestButton={
-                                    <EntityTestButton
-                                        disabled={!hasConnectors}
-                                        logEvent={CustomEvents.CAPTURE_TEST}
-                                    />
-                                }
-                                SaveButton={
-                                    <EntitySaveButton
-                                        disabled={!draftId}
-                                        taskNames={tasks}
-                                        logEvent={CustomEvents.CAPTURE_CREATE}
-                                    />
-                                }
-                            />
-                        }
-                        RediscoverButton={
-                            <RediscoverButton
-                                entityType={entityType}
-                                disabled={!hasConnectors}
-                                postGenerateMutate={updateDraftSpecs}
-                            />
-                        }
-                    />
+                    <MutateDraftSpecProvider value={updateDraftSpecs}>
```

Comment:
This kind of change is in several files. The main change is a new wrapper for the context and no longer passing this in props.